actors/miner/tests/util.rs

```diff
@@ -1183,6 +1198,27 @@ pub fn assert_bitfield_equals(bf: &BitField, bits: &[u64]) {
     assert!(bf == &rbf);
 }
 
+#[allow(dead_code)]
+pub fn make_empty_bitfield() -> UnvalidatedBitField {
+    UnvalidatedBitField::Validated(BitField::new())
+}
+
+#[allow(dead_code)]
+pub fn make_bitfield(bits: &[u64]) -> UnvalidatedBitField {
+    let mut bf = BitField::new();
+    for bit in bits {
+        bf.set(*bit);
+    }
+    UnvalidatedBitField::Validated(bf)
+}
+
+fn get_bitfield(ubf: &UnvalidatedBitField) -> BitField {
+    match ubf {
+        UnvalidatedBitField::Validated(bf) => bf.clone(),
+        UnvalidatedBitField::Unvalidated(bytes) => BitField::from_bytes(&bytes).unwrap(),
+    }
+}
```

Comment:
Can't you just call `validate_mut` at the callsite?

Reply:
no, borrow checker doesn't let me and it becomes really messy to accomodate him; so after spending a good 10min trying various combinations to make the beast happy I gave up and just did this.